Gemfile

```diff
@@ -15,8 +15,7 @@ end
 # only the master branch is supported on rspec-support
 gem "rspec-support", :git => "git://github.com/rspec/rspec-support.git"
 
-gem "cucumber", "~> 1.1.9"
-gem "aruba",    "~> 0.5"
+gem "aruba"
```

Comment:
FWIW, for the other RSpec gems, we haven't bothered to try to keep up to date on aruba.  The API has been a bit of a moving target and I spent a bunch of time trying to get current and then decided it wasn't worth it.  Might be worth sticking to an old version here as well.